SU2_CFD/src/solvers/CRadP1Solver.cpp

```diff
@@ -282,6 +282,14 @@ void CRadP1Solver::BC_Isothermal_Wall(CGeometry *geometry, CSolver **solver_cont
   /*--- Get the specified wall emissivity from config ---*/
   Wall_Emissivity = config->GetWall_Emissivity(Marker_Tag);
 
+  /*--- Clamp emissivity to valid physical range [0,1] to prevent division by zero ---*/
+  if (Wall_Emissivity < 0.0) {
+    Wall_Emissivity = 0.0;
+  }
+  if (Wall_Emissivity > 1.0) {
+    Wall_Emissivity = 1.0;
+  }
+
```

Comment:
We shouldn't silently clip this but quit with an error when emissivity is unphysical. This can be done in cconfig.cpp